tests/e2e/sample-applications/virt-todo/test.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+from datetime import datetime
+import json
+import requests
+
+
+#base_url = "http://localhost:8000"
```

Comment:
maybe remove this one.

Reply:
saying sothing, just to put my name on this PR :P

Reply:
removed all of them :)